x-pack/plugins/spaces/public/lib/constants.ts

```diff
@@ -4,8 +4,14 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
 import chrome from 'ui/chrome';
 
-export const SPACES_FEATURE_DESCRIPTION = `Organize your dashboards and other saved objects into meaningful categories.`;
+export const SPACES_FEATURE_DESCRIPTION = i18n.translate(
```

Comment:
Based on the demo we had yesterday, I thought it was advised not to perform translations as part of static imports, but rather to wrap it in a function. Is it ok that we're doing this statically?

Reply:
Yes, for the client side this translation code will work, but on the server side it won't work. In any case to be on the safe side and to not relay on the file loading order we should wrap statically defined constants into function. I updated code. Thank you for notice.